server/errors.c

```diff
@@ -31,7 +31,7 @@ static int do_error(const char *func, DWORD err)
 	char *buffer;
 
 	buffer = NULL;
-	if (FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM
+	if (FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM
```

Comment:
Why did you change to `FormatMessage`? If I understand Windows API correctly, `FormatMessage` requires a `LPTSTR` buffer (which is either `char *` or `WCHAR *`), but `error` only supports `char *`, not wide chars. Did it work with `FormatMessageW`?